modules/router-store/src/serializers/minimal_serializer.ts

```diff
@@ -0,0 +1,53 @@
+import { RouterStateSnapshot, ActivatedRouteSnapshot } from '@angular/router';
+import { BaseRouterStoreState, RouterStateSerializer } from './shared';
+
+export interface MinimalActivatedRouteSnapshot {
+  routeConfig: ActivatedRouteSnapshot['routeConfig'];
```

Comment:
Is there a better way to type this?

Reply:
Other than to use its actual type `Route`?

Reply:
Yea, I tried to just use `ActivatedRouteSnapshot` and override some properties but I'm not sure if this is possible.

Reply:
I'm not sure either